src/nspawn/nspawn-mount.c

```diff
@@ -187,7 +189,7 @@ static int tmpfs_patch_options(
 
         char *buf = NULL;
 
-        if (userns && uid_shift != 0) {
+        if (userns) {
```

Comment:
hmm, why did you drop this check? why add these options if uid= and gid= are 0 anyway?